cypress/support/views.ts

```diff
@@ -63,7 +63,10 @@ declare global {
 }
 
 Cypress.Commands.add("viewRoomByName", (name: string): Chainable<JQuery<HTMLElement>> => {
-    return cy.findByRole("treeitem", { name: name }).should("have.class", "mx_RoomTile").click();
+    return cy
+        .findByRole("treeitem", { name: new RegExp("^" + name) })
+        .should("have.class", "mx_RoomTile")
```

Comment:
Sure will update docs, this is because the room tile has notification test appended to it (unlike space tiles) so its anchored to ignore it

![image](https://github.com/matrix-org/matrix-react-sdk/assets/2403652/961543ba-f1e6-47bd-9c86-3a0cd483b35f)